api/comments.rb

```diff
@@ -1,5 +1,5 @@
 get "#{APIPREFIX}/comments/:comment_id" do |comment_id|
-  comment.to_hash(recursive: bool_recursive).to_json
+  comment(nil, true).to_hash(recursive: bool_recursive).to_json
```

Comment:
Also, it should be simple to add a test to show that if you force the count to be wrong, this will fix it.